src/SprykerEco/Zed/Payone/Business/Payment/MethodSender/PayoneDebitMethodSender.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Payone\Business\Payment\MethodSender;
+
+use Generated\Shared\Transfer\DebitResponseTransfer;
+use Generated\Shared\Transfer\PayoneStandardParameterTransfer;
+use Orm\Zed\Payone\Persistence\SpyPaymentPayoneApiLog;
+use SprykerEco\Zed\Payone\Business\Api\Adapter\AdapterInterface;
+use SprykerEco\Zed\Payone\Business\Api\Response\Container\DebitResponseContainer;
+use SprykerEco\Zed\Payone\Business\Api\Response\Mapper\DebitResponseMapper;
+use SprykerEco\Zed\Payone\Business\Payment\DataMapper\StandartParameterMapperInterface;
+use SprykerEco\Zed\Payone\Business\Payment\PaymentMapperReader;
+use SprykerEco\Zed\Payone\Persistence\PayoneQueryContainerInterface;
+
+class PayoneDebitMethodSender extends AbstractPayoneMethodSender implements PayoneDebitMethodSenderInterface
+{
+    /**
+     * @var \SprykerEco\Zed\Payone\Business\Api\Adapter\AdapterInterface
+     */
+    protected $executionAdapter;
+
+    /**
+     * @var \SprykerEco\Zed\Payone\Persistence\PayoneQueryContainerInterface
+     */
+    protected $queryContainer;
+
+    /**
+     * @var \Generated\Shared\Transfer\PayoneStandardParameterTransfer
+     */
+    protected $standardParameter;
+
+    /**
+     * @var \SprykerEco\Zed\Payone\Business\Payment\DataMapper\StandartParameterMapperInterface
+     */
+    protected $standartParameterMapper;
+
+    /**
+     * @param \SprykerEco\Zed\Payone\Business\Api\Adapter\AdapterInterface $executionAdapter
+     * @param \SprykerEco\Zed\Payone\Persistence\PayoneQueryContainerInterface $queryContainer
+     * @param \Generated\Shared\Transfer\PayoneStandardParameterTransfer $standardParameter
+     * @param \SprykerEco\Zed\Payone\Business\Payment\DataMapper\StandartParameterMapperInterface $standartParameterMapper
+     * @param \SprykerEco\Zed\Payone\Business\Payment\PaymentMapperReader $paymentMapperReader
+     */
+    public function __construct(
+        AdapterInterface $executionAdapter,
+        PayoneQueryContainerInterface $queryContainer,
+        PayoneStandardParameterTransfer $standardParameter,
+        StandartParameterMapperInterface $standartParameterMapper,
+        PaymentMapperReader $paymentMapperReader
+    ) {
+        parent::__construct($queryContainer, $paymentMapperReader);
+        $this->executionAdapter = $executionAdapter;
+        $this->queryContainer = $queryContainer;
```

Comment:
Not needed.

Reply:
Used in line 97 
return $this->queryContainer->createPaymentByTransactionIdQuery($transactionId)->findOne();